src/dags/importscripts/import_processenverbaalverkiezingen.py

```diff
@@ -0,0 +1,134 @@
+import csv
+import ftplib
+from more_ds.network.url import URL
+from os import path as ospath
+from typing import List, TypedDict, Tuple, Optional, Generator
+
+
+class Data(TypedDict):
+    """Datastructure processenverbaal verkiezingen"""
+
+    verkiezingsjaar: int
+    volgnummer: str
+    uri: str
+    documentnaam: str
+    stemlocatie: str
+    id: str
+
+
+class FTPListing:
+    """This class is used for traversing a given folder base on a FTP
+    server (objectstore) using BFS (Breadth-First search) algorithm.
+    """
+
+    def __init__(self, connection: ftplib.FTP) -> None:
+        """Initialize
+
+        Args:
+            Connection: Hold the connection to the objectstore
+
+        """
+        self.connection = connection
+
+    def list_dirs_and_files(self, _path: str) -> Tuple[List[str], Optional[List[str]]]:
+        """Return files and directories within a path
+        So it can be used to identifiy if there is need for another directory scan
+        to locate files. Files already found can be used to get the URI.
+
+        Args:
+            _path: The path for looking for files and directories (within the same path)
+
+        Returns:
+            list of found dictories and files
+
+        """
+
+        list_, dirs, files = [], [], []
```

Comment:
I'd put these closer to their use site, inside the else block.

Reply:
Adjusted